From 5358f24b8ba5c359db504421d09fc16cf9e7f8cf Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 11 Dec 2019 18:01:26 +0000
Subject: [PATCH] setup-apkrepos: use cdn as default answer

use '1', which should be dl-cdn, as the default answer
---
 setup-apkrepos.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup-apkrepos.in b/setup-apkrepos.in
index 81e768e..1b05978 100644
--- a/setup-apkrepos.in
+++ b/setup-apkrepos.in
@@ -175,7 +175,7 @@ if [ -r "$APKREPOS_PATH" ]; then
 fi
 
 get_alpine_release
-default_answer=f
+default_answer=1
 changed=false
 
 if [ $# -gt 0 ]; then
-- 
2.24.1

